### PR TITLE
Add rtgtools

### PR DESCRIPTION
**Pull request by ramprasadn**
## PR checklist

Addresses https://github.com/nf-core/raredisease/issues/250 

- [ ] This comment contains a description of changes (with reason).
- [ ] If you've fixed a bug or added code that should be tested, add tests!
- [ ] If you've added a new tool - have you followed the pipeline conventions in the [contribution docs](https://github.com/nf-core/raredisease/tree/master/.github/CONTRIBUTING.md)
- [ ] If necessary, also make a PR on the nf-core/raredisease _branch_ on the [nf-core/test-datasets](https://github.com/nf-core/test-datasets) repository.
- [ ] Make sure your code lints (`nf-core lint`).
- [ ] Ensure the test suite passes (`nextflow run . -profile test,docker --outdir <OUTDIR>`).
- [ ] Ensure the test suite passes (`nextflow run . -profile test_one_sample,docker --outdir <OUTDIR>`).
- [ ] Usage Documentation in `docs/usage.md` is updated.
- [ ] Output Documentation in `docs/output.md` is updated.
- [ ] `CHANGELOG.md` is updated.
- [ ] `README.md` is updated (including new tool citations and authors/contributors).

### CODE REVIEW

File: subworkflows/local/variant_evaluation.nf [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+
+        BCFTOOLS_REHEADER (ch_rtgvcfs_dbs, [[:],[]])
+
+        TABIX_TRUTHVCF (BCFTOOLS_REHEADER.out.vcf)
+
+        BCFTOOLS_REHEADER.out.vcf
+            .join(TABIX_TRUTHVCF.out.tbi)
+            .set { ch_truthvcf_tbi }
+
+        ch_snv_vcf_tbi
```
Comment (jemten) [inline subworkflows/local/variant_evaluation.nf:35]: Don't we need to split `ch_snv_vcf_tbi` into sample vcfs?

**Review by jemten — COMMENTED** [review]
some comments and questions :)

**Comment by jemten** [comment]
Could you also update the documentation on how to use this

File: subworkflows/local/variant_evaluation.nf (hunk 1 above)
Comment (ramprasadn) [reply subworkflows/local/variant_evaluation.nf:35]: Not necessary actually.. specifying --samples <samplename> in the command takes care of that :)

Comment (jemten) [reply subworkflows/local/variant_evaluation.nf:35]: oh nice!

File: subworkflows/local/prepare_references.nf [hunk 2]
```diff
@@ -129,7 +133,7 @@ workflow PREPARE_REFERENCES {
         genome_dict           = GATK_SD.out.dict.collect()                                       // channel: [ path(dict) ]
         readcount_intervals   = Channel.empty()
                                     .mix(ch_preprocwgs.interval_list,ch_preprocwes.interval_list)// channel: [ path(intervals) ]
-
+        sdf                   = RTGTOOLS_FORMAT.out.sdf                                          // channel: [ val (meta), path(intervals) ]
```
[the remark this answers is not in the document]
Comment (ramprasadn) [reply subworkflows/local/prepare_references.nf:136]: Good point! Fixed 👍🏻

**Review by jemten — APPROVED** [review]
Looks good!

**Pull request merged**